src/keylessBackup/KeylessBackupProgress.tsx

```diff
@@ -35,9 +49,44 @@ function KeylessBackupProgress({
     return () => BackHandler.removeEventListener('hardwareBackPress', backPressListener)
   }, [])
 
-  // TODO(ACT-781): Implement Restore flow designs
   if (route.params.keylessBackupFlow === KeylessBackupFlow.Restore) {
-    return <></>
+    switch (keylessBackupStatus) {
+      case KeylessBackupStatus.NotStarted:
```

Comment:
Is it possible for `keylessBackupStatus === KeylessBackupStatus.NotStarted` here? Not obvious to me why an in-progress section would show when the status is `NotStarted`, or maybe I'm missing something on how the status is tracked.